Bottom: f7c2b2dc7a5a1dd627be0e2d0ca7b9ffa9521d72
Top:    42a124064cbe96585b4e76f2f802bd5a7cb7358b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-28 00:38:00 +0100

Refresh of crosshair-c-nasty-thing-fixme

---

diff --git a/src/crosshair.c b/src/crosshair.c
index f10a250..ca51f4f 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -692,10 +692,8 @@ CrosshairOn (void)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
-#if 0
       DrawAttached ();
       DrawMark ();
-#endif
     }
 }
 
@@ -708,10 +706,8 @@ CrosshairOff (void)
   if (Crosshair.On)
     {
       Crosshair.On = false;
-#if 0
       DrawAttached ();
       DrawMark ();
-#endif
     }
 }
